airbyte-integrations/connectors/source-declarative-manifest/unit_tests/fixtures/__init__.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+import pytest
+import os
+import yaml
+
+def get_fixture_path(file_name):
+    return os.path.join(os.path.dirname(__file__), file_name)
+
+@pytest.fixture
```

Comment:
@lmossman Just tagging you again here as the tests for this were accidently were committed to an upstream branch.

just in case you wanted to review :)